src/containers/shared/components/Transaction/OracleDelete/Simple.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+
+export const Simple: TransactionSimpleComponent = ({
+  data,
+}: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { OracleDocumentID } = data.instructions
+  return (
+    <div className="oracle-delete">
+      {
+        // OracleDocumentID could be zero.
+        OracleDocumentID !== null && (
```

Comment:
```suggestion
        OracleDocumentID != null && (
```
`null` is special in that it doesn't need the extra `=`
https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons

Reply:
Actually, is this check even needed if the transaction enforces that it's always included?